src/components/Hero/ContentHero/index.tsx

```diff
@@ -1,42 +1,22 @@
-import { Box, Heading, SimpleGrid, Stack, Text } from "@chakra-ui/react"
-
 import type { CommonHeroProps } from "@/lib/types"
 
 import Breadcrumbs from "@/components/Breadcrumbs"
-import { Image } from "@/components/Image"
+import { TwImage } from "@/components/Image"
+import { Stack } from "@/components/ui/flex"
 
 import { CallToAction } from "../CallToAction"
 
 export type ContentHeroProps = Omit<CommonHeroProps<string>, "header">
 
 const ContentHero = (props: ContentHeroProps) => {
-  const {
-    breadcrumbs,
-    heroImg,
-    buttons,
-    title,
-    description,
-    blurDataURL,
-    maxHeight,
```

Comment:
Removed this prop as it was not being used and was a bit hacky to set up with tw.